### PR TITLE
Add DSDT RHP device table for MX6ULL platform.

### PR DESCRIPTION
**Pull request by henie**

### CODE REVIEW

File: Platform/NXP/iMX6ULL_EVK_512MB/AcpiTables/Dsdt-Rhp.asl [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+  {    
+    ToUUID("daffd814-6eba-4d8c-8a91-bc9bbf4aa301"),
+    Package()
+    {
+      // GPIO Pin Count and supported drive modes
+      Package (2) { "GPIO-PinCount", 124 },
+      Package (2) { "GPIO-UseDescriptorPinNumbers", 1 },
+
+      // InputHighImpedance, InputPullUp, OutputCmos
+      Package (2) { "GPIO-SupportedDriveModes", 0x0b },
```
Comment (jordanrh1) [inline Platform/NXP/iMX6ULL_EVK_512MB/AcpiTables/Dsdt-Rhp.asl]: The GPIO controller supports pull down, so this should be 0xf

File: Platform/NXP/iMX6ULL_EVK_512MB/AcpiTables/Dsdt-Platform.asl [hunk 2]
```diff
@@ -13,7 +13,7 @@
 *
 */
 
-OperationRegion(GLBL,SystemMemory,0x10817000,0x10)
+OperationRegion(GLBL,SystemMemory,0x80817000,0x10)
```
Comment (jordanrh1) [inline Platform/NXP/iMX6ULL_EVK_512MB/AcpiTables/Dsdt-Platform.asl:16]: >0x80817000 [](start = 34, length = 10)

Can you confirm that this is same address passed to 'globalpage' command, and that this address is not overlapping any other address in the memory map?

**Comment by jordanrh1** [comment]
The way to test this change is to build [GpioTestTool.exe](https://github.com/Microsoft/Windows-iotcore-samples/tree/develop/BusTools/GpioTestTool) and run 
```
gpiotesttool -list
```
and verify that the expected pins are listed, then run
```
gpiotesttool 25
> info
> setdrivemode output
> write 0
> write 1
```
And verify that pin changes state as expected. There are also commands to test interrupts and all drive modes. Run `help` to see all supported commands of gpiotesttool.

**Review by jordanrh1 — CHANGES_REQUESTED** [review]
Please address comments.

File: Platform/NXP/iMX6ULL_EVK_512MB/AcpiTables/Dsdt-Platform.asl (hunk 2 above)
Comment (henie) [reply Platform/NXP/iMX6ULL_EVK_512MB/AcpiTables/Dsdt-Platform.asl:16]: Yes, this address 0x80817000 is correct one, 0x10817000 is not valid address region for 6ULL

**Comment by henie** [comment]
GPIO test tool test result

GpioTestTool.exe -list
The default GPIO controller has 124 pins:
  Pin 25 is available

GpioTestTool.exe 25
Type 'help' for a list of commands
        Pin Number: 25
      Sharing Mode: Exclusive
  Debounce Timeout: 0 ms
        Drive Mode: inputPullUp
             Value: High
Set drive mode as Output OK
Write pin as 0 OK
Write pin as 1 OK

**Review by jordanrh1 — APPROVED** [review]

**Pull request merged**